app/views/pages/index.html.haml

```diff
@@ -117,7 +117,7 @@
         .feature
           = graphical_icon "features-do-not-know"
           %h3.text-h4 You don’t know what you don’t know
-          %p.text-p-base When you're learning a new language, the hardest thing is discovery of knowledges. Our mentors can look at your code and immediately see the ideas you're not familiar with and give you a step up by unlocking new knowledge.
+          %p.text-p-base When you're learning a new language, it can be challenging to solve a problem when you're not aware of gaps in your knowledge. Our mentors can look at your code and immediately see the concepts you're not familiar with and give you a step up by unlocking new knowledge.
```

Comment:
Thanks for this.

Two things:
1. It can't get any longer than it is. It needs to fit onto 4 lines.
2. I deliberately didn't use "concepts" as it's an overloaded term in Exercism

Maybe
```suggestion
          %p.text-p-base When learning a new language, the hardest part is not being aware of the gaps in your knowledge. Our mentors can look at your code and immediately see the ideas you're not familiar with and give you a level-up by unlocking new knowledge.
```